Step 1: Define Hypotheses
The first step is to set up the null and alternative hypotheses:

Null Hypothesis (H0): The group's mean
systolic blood pressure is not significantly different from the regular population (μ = 120 mm Hg).
Alternative Hypothesis (H1): The group's mean systolic blood pressure is significantly different from the regular population.

Step 2: Choose Significance Level
α = 0.05

In [1]:
import math

sample_mean = 130.1
population_mean = 120
sample_std = 21.21
sample_size = 100

test_statistic = (sample_mean - population_mean) / (sample_std / math.sqrt(sample_size))
print("Test Statistic:", test_statistic)


Test Statistic: 4.761904761904759


with a test statistic of 4.761904761904759 and a significance level of 0.05, the decision would be to reject the null hypothesis and conclude that the group is significantly different from the regular population in terms of systolic blood pressure